src/cargo/util/network/retry.rs

```diff
@@ -47,6 +47,7 @@ use anyhow::Error;
 use rand::Rng;
 use std::cmp::min;
 use std::time::Duration;
+use time::OffsetDateTime;
```

Comment:
I feel like we are on the way of dropping `time`? https://github.com/rust-lang/cargo/pull/15293

Reply:
Maybe using jiff isn't the right call?

In my testing, it can take 2 to 10 seconds for jiff to initialize.

I suspect part of the problem is that jiff is configured to use the system timezone data, and it's probably loading and initializing a bunch of junk. Unfortunately because `gix-date` enables the default features, we can't easily disable that AFAIK.

Can we maybe get gix-date to not use the default? I don't fully understand what all the jiff features do, or when or if they are necessary.

Reply:
It seems like `jiff` init is taking around that time in CI, which I agree is unacceptable. I'm not seeing the delay on Windows, but that's likely because jiff isn't using system timezone data there.

Reply:
I wonder if there's a way to do the parsing without loading the timezone data. The HTTP Date header is always GMT.